cli/src/opts/wallet/multi_wallet.rs

```diff
@@ -386,6 +395,28 @@ impl MultiWallet {
         Ok(None)
     }
 
+    pub async fn aws_signers(&self, chain_id: u64) -> Result<Option<Vec<AwsSigner>>> {
+        if self.aws {
+            let mut wallets = vec![];
+            let client =
+                AwsClient::new_with(AwsChainProvider::default(), AwsHttpClient::new().unwrap());
+
+            let kms = KmsClient::new_with_client(client, AwsRegion::default());
```

Comment:
Should the region be configurable?

Reply:
default region reads from the standard AWS env locations

https://rusoto.github.io/rusoto/rusoto_core/region/enum.Region.html#default